src/test/java/step3/CarTest.java

```diff
@@ -0,0 +1,37 @@
+package step3;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import step3.common.vo.Car;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class CarTest {
+
+    private static Car car;
```

Comment:
```suggestion
    private Car car;
```

인스턴스 변수로 구현하는 것은 어떨까?